test/functional/change_stream_spec.test.js

```diff
@@ -2,13 +2,30 @@
 
 const path = require('path');
 const chai = require('chai');
-const loadSpecTests = require('../spec').loadSpecTests;
+const { loadSpecTests } = require('../spec');
+const { runUnifiedTest } = require('./unified-spec-runner/runner');
 const camelCase = require('lodash.camelcase');
-const setupDatabase = require('./shared').setupDatabase;
-const delay = require('./shared').delay;
+const { setupDatabase } = require('./shared');
+const { delay } = require('./shared');
 const expect = chai.expect;
 
-describe('Change Stream Spec', function () {
+describe('Change Streams Spec - Unified', function () {
+  for (const changeStreamTest of loadSpecTests(path.join('change-streams', 'unified'))) {
+    expect(changeStreamTest).to.exist;
+    context(String(changeStreamTest.description), function () {
+      for (const test of changeStreamTest.tests) {
+        it(String(test.description), {
+          metadata: { sessions: { skipLeakTests: true } },
+          test: async function () {
+            await runUnifiedTest(this, changeStreamTest, test);
+          }
+        });
+      }
+    });
+  }
+});
```

Comment:
```suggestion
describe('Change Streams Spec - Unified', function () {
  runUnifiedSuite(loadSpecTests(path.join('change-streams', 'unified')));
});
```

We've been bringing this block of code to each new unified spec when we already have a shared way of doing it, IIRC it came in a merge after the original unified runner so we've been using this pattern as the template. Anyway! this works if we wanna pair down the diff a bit :)

Reply:
Reducing boilerplate is always good, thanks for pointing out the existence of this helper! 👍